src/platform/input/game-pads.js

```diff
@@ -133,28 +152,28 @@ class GamePads {
         if (pads.length > 0) {
             pads.length = 0;
         }
-
-        if (this.gamepadsSupported) {
-            const padDevices = navigator.getGamepads ? navigator.getGamepads() : navigator.webkitGetGamepads();
-            for (let i = 0, len = padDevices.length; i < len; i++) {
-                if (padDevices[i]) {
-                    pads.push({
-                        map: this.getMap(padDevices[i]),
-                        pad: padDevices[i]
-                    });
-                }
+        const padDevices = getGamepads();
+        for (let i = 0, len = padDevices.length; i < len; i++) {
+            if (padDevices[i]) {
+                pads.push({
+                    map: this.getMap(padDevices[i]),
+                    pad: padDevices[i]
+                });
             }
         }
         return pads;
     }
 
+    /**
+     * @param {Gamepad} pad - The HTML5 Gamepad object.
+     * @returns {ButtonsAxes} Object defining the order of buttons and axes for given HTML5 Gamepad.
+     */
     getMap(pad) {
         for (const code in PRODUCT_CODES) {
-            if (pad.id.indexOf(code) >= 0) {
+            if (pad.id.includes(code)) {
```

Comment:
`Array#includes` is not available for IE11, hence the use of `indexOf`.

Reply:
Hi @willeastcott, this isn't `Array#includes` but `String#includes` and it's polyfilled and used in other places of the engine aswell. Thank you for taking your time to review, I appreciate it a lot!